pkg/util/pod.go

```diff
@@ -21,6 +21,13 @@ import (
 	"strings"
 )
 
+func PodReadyStatus(pod *corev1.Pod) bool {
+	if pod.Status.Phase == corev1.PodRunning && !podNotRunning(pod.Status.ContainerStatuses) {
```

Comment:
If containerReady is not true, this function will return true, it dose not meet expectations. You can use IsPodReady function instead